src/Our.Umbraco.Ditto/ComponentModel/Attributes/DittoLazyAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Our.Umbraco.Ditto.ComponentModel.Attributes
+{
+    /// <summary>
+    /// The Ditto lazy property attribute.
+    /// Used for specifying that Ditto should lazy load this property.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Class, AllowMultiple = false)]
```

Comment:
I think there is a confusion I need to check here. Having the attribute declared on a type currently makes all of it's contained virtual properties lazy, so it could make sense to use on an Interface (assuming you can declare properties as virtual on an interface) but wouldn't so much on an Enum.

I did have this discussion with Lee, but would be good to get your view point. What are you expecting the class level Lazy attribute to do? Make all it's inner properties lazy? or make the parent declaring property lazy? (ie, the parent property that defines it's type as your type with the lazy attribute applied). If it's the later, how would this affect enumerable props?